frontend/src/scenes/trends/LineGraph.js

```diff
@@ -22,27 +22,62 @@ export class LineGraph extends Component {
 
     buildChart = () => {
         const myChartRef = this.chartRef.current.getContext('2d')
-        const { datasets, labels, options } = this.props
+        let { datasets, labels, options } = this.props
 
         if (typeof this.myLineChart !== 'undefined') this.myLineChart.destroy()
         let colors = ['blue', 'orange', 'green', 'red', 'purple', 'gray']
         let getVar = variable => getComputedStyle(document.body).getPropertyValue('--' + variable)
         const _this = this
-
-        this.myLineChart = new Chart(myChartRef, {
-            type: this.props.type || 'line',
-            data: {
-                //Bring in data
-                labels: labels,
-                datasets: datasets.map((dataset, index) => ({
+        datasets = [
```

Comment:
this could be condensed but i opted for least intrusive edit for the moment to just get the UI in

Reply:
cleaned up